pgpdump/packet.py

```diff
@@ -532,12 +539,11 @@ def parse_key_material(self, offset):
             offset = self.parse_oid_data(offset)
             #self.point_q, offset = get_mpi(self.data, offset)
             # TODO Look for specifics of curve25519 if any
-        elif 100 <= self.raw_pub_algorithm <= 110:
-            # Private/Experimental algorithms, just move on
-            pass
         else:
-            raise PgpdumpException("Unsupported public key algorithm %d" %
```

Comment:
I am not sure, if I wouldn't prefer an exception...
Why do you think just passing is preferable?

Reply:
The code does not raise an exception for private/experimental algorithms, but for unknown ones. Both it can not handle properly, while the only difference between these two cases is: some numbers are "reservered" while others are simply unused.